mmv1/products/securitycenterv2/product.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright 2019 Google Inc.
```

Comment:
Nit: I would actually change this to 2024 (looked like maybe you had this previously)